### DISCUSSION TITLE
Project Delivery API Forms Scope

### DISCUSSION THREAD

**Discussion by cjwb486** [iTwin/itwinjs-core#1322] [Q&A] [answered]
I'm trying to pull forms data from a connected project, the forms API requires a registered app with 'forms: read' & 'forms: modify' scope.

When I register a new Project Delivery app though I only get the option to add scope for: Issues, Validation & Projects. 

I understand this is in tech preview still so may not be available yet, just wasn't sure if I was missing something?

**Comment by calebmshafer** [comment, marked as answer]
@cjwb486 can you try going to your App details and under the "Project Delivery" section click "Manage your scopes"? The `issues:read` and `issues:modify` scopes should now be available for you to add.

It looks like there has been an update since you last tried to do it and they should now be added by default with the "Project Delivery" API Association as well.
